dpath/__init__.py

```
@@ -27,6 +27,10 @@
 from dpath.exceptions import InvalidKeyName, PathNotFound
 from dpath.types import MergeType, PathSegment, Creator, Filter, Glob, Path, Hints
 
+import sys
```

Move up to standard library imports

---

Sys disappeared since we raise exception instead of printing